proxy_test.go

```diff
@@ -900,3 +901,29 @@ func TestHyperstartResponse(t *testing.T) {
 
 	rig.Stop()
 }
+
+func TestStoreRestore(t *testing.T) {
+	storeStateDir = "/tmp/clearcontainers/proxy/"
+	rig := newTestRig(t)
+	rig.Start()
+
+	// clean up a possible state
+	os.RemoveAll(storeStateDir)
+	assert.Equal(t, rig.proxy.restoreState(), false)
```

Comment:
Not a major problem, but in the runtime for new code we've started creating a `assert` variable to avoid having to hass the `testing.T` each time:

```
assert := assert.New(t)
  :
  :
assert.Equal(rig.proxy.restoreState(), false)
```

Reply:
Got it. Thanks. Will do.